app/home/static/js/script-v2.js

```diff
@@ -317,22 +317,27 @@ function alert_message(type = 'success', message = 'Success!', icon = '', time =
 */
 
 function add_max_length_counter() {
-    let inputGroup = $("[maxlength]").parent()
+    let inputGroup = $("[maxlength]").parent();
+
+    for (const element of inputGroup) {
+        let label = $($(element).children()[0]);
```

Comment:
You const ALWAYS. Should be the default option to declare a variable. Use let only when it needs to be mutated.